src/generators/asset-types/__tests__/woff2.ts

```diff
@@ -10,9 +9,7 @@ jest.mock('ttf2woff2', () =>
 );
 
 const mockOptions = (woffOptions = { __mock: 'options__' } as any) =>
-  (({
-    formatOptions: { [FontAssetType.WOFF2]: woffOptions }
-  } as unknown) as FontGeneratorOptions);
+  (({} as unknown) as FontGeneratorOptions);
```

Comment:
Why are we removing the woff options mock from tests? They check that the generator options are correctly passed to the font generator.

Reply:
Hello @tancredi 

Fiddling in these @types/ definitions I discovered that ttf2woff2 does not have such options. For unknown rason it used to work, but anyhow, the lib will not process and do anything with it. So I removed the options from the font options type, and this spec conflicted with that fix. It expected the non-existent font option to be passable.

Reply:
@tancredi 

Yes, I verified that now. It is definiteley because I found out taht ttf2woff2 simply does not have any such second param.

See:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ttf2woff2/index.d.ts#L8

And in TTF2WOFF2 itself:

https://github.com/nfroidure/ttf2woff2/blob/master/jssrc/index.js#L5

No second param there...

Reply:
That's great to know! I've removed some other unnecessary format options from the types! cheers